tests/desktop/developer_hub/test_submit_fxos_add_on.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+import shutil
+import uuid
+import warnings
+from zipfile import ZipFile
+
+import pytest
+
+from pages.desktop.developer_hub.content_tools import ContentTools
+from tests.base_test import BaseTest
+
+
+class TestSubmitFxOSAddOn(BaseTest):
+
+    @pytest.fixture
+    def add_on(self, request, tmpdir):
+
+        add_on_name = str(uuid.uuid4())
+        new_zip_path = str(tmpdir.join('%s.zip' % add_on_name))
+
+        # copy zip file to tmpdir
+        shutil.copyfile(self._get_resource_path('fxosaddon.zip'), new_zip_path)
+
+        # update manifest.json in zip file
+        with ZipFile(new_zip_path, 'a') as zipfile:
+            data = json.loads(zipfile.read('manifest.json'))
+            data['name'] = add_on_name
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
```

Comment:
When we add the `manifest.json` file back into the zip file, and it overwrites the existing file, a warning is issued. It's not harmful, but I thought it might be distracting in the output. Adding these two lines prevents that warning from being issued. I can remove them, as the warning is just a warning, but I thought this was nice/cleaner for the output.